### ISSUE TITLE
The Crew 2 (646910)

### ISSUE THREAD

**Issue by BarrelKnight723** [ValveSoftware/Proton#3650]
I've been owning the game The Crew 2 and I think now is the time I need to talk about it. When I launch the game it gets stuck on "Starting Battleye Service". Can anyone help me? I am on Pop OS 19.10

## System Information
- GPU:  AMD Radeon RX580
- Driver/LLVM version:  Mesa 20.0
- Kernel version:  5.3
- Proton version: 5.0-5

**Comment by kisak-valve** [comment]
Hello @CSForever07, please grab the [compatibility report template](https://github.com/ValveSoftware/Proton/issues/new?template=compatibility-report.md), copy it to your opening post, then fill it out.

Unfortunately, BattlEye currently does not work with Proton, so the information is unlikely to make a difference one way or another unless we hear about some progress with that.

**Comment by AsciiWolf** [comment]
Why did you close the ticket?

**Comment by BarrelKnight723** [comment]
With the news of battleye coming to Linux I figured that this issue would be resolved sooner or later.

**Comment by AsciiWolf** [comment]
True, but it would still be nice to keep this game compatibility ticket opened.

**Comment by BarrelKnight723** [comment]
I have reopened this issue now. Hopefully it will be convenient some other time.

**Comment by pollux78** [comment]
damn im surprised this game has gotten battleye enabled for linux, very sadge as its on sale

**Comment by lzq960605** [comment]
The Crew Motorfest  It's already playable, how long will The Crew 2 take?
